RestImageResize/web.config.transform

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <appSettings>
     <add key="RestImageResize.DefautTransform" value="downFit"/>
+    <add key="RestImageResize.PrivateKeys" value="" />
```

Comment:
Then all the settings should be commented ;) Now we have everything in readme - so this should be another pull request.